examples/persistence_service/library_api/c/persistence_service_library_api.c

```diff
@@ -0,0 +1,135 @@
+/*******************************************************************************
+ (c) 2023 Copyright, Real-Time Innovations, Inc. All rights reserved.
+ RTI grants Licensee a license to use, modify, compile, and create derivative
+ works of the Software. Licensee has the right to distribute object form only
+ for use with RTI products. The Software is provided "as is", with no warranty
+ of any type, including any warranty for fitness for any purpose. RTI is under
+ no obligation to maintain or support the Software. RTI shall not be liable for
+ any incidental or consequential damages arising out of the use or inability to
+ use the software.
+ ******************************************************************************/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "ndds/ndds_c.h"
+#include "persistence/persistence_service.h"
+
+int service_shutdown(struct RTI_PersistenceService *service)
+{
+    DDS_Boolean ok = DDS_BOOLEAN_FALSE;
+
+    if (service != NULL) {
+        ok = RTI_PersistenceService_stop(service);
```

Comment:
```suggestion
        DDS_Boolean ok = RTI_PersistenceService_stop(service);
```

Reply:
You can reduce the scope of that variable

Reply:
Done.